src/components/BButton/BButton.vue

```diff
@@ -1,18 +1,75 @@
 <template>
-  <button>{{text}}</button>
+  <button
+    :class="computedClass"
+    :disabled="disabled"
```

Comment:
Could also do `v-bind="{disabled, type, aria-label}"`. Don't know if we have a preference there yet

Reply:
tried that and I was getting an error

Reply:
Huh, that's surprising